src/layouts/Default/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+
+const layoutClassName = 'layout-default';
+
+export const layoutDefaultTestIds = {
+  base: layoutClassName,
+};
+
+interface LayoutProps {
+  children: React.ReactNode;
+}
+
+const LayoutDefault = ({ children }: LayoutProps) => {
+  useEffect(() => {
+    document.body.classList.add(layoutClassName);
+  });
```

Comment:
If you agree with the previous comment please also add it here

```suggestion
  }, []);
```

Reply:
Added https://github.com/Lahphim/react-surveyor-challenge/pull/55/commits/d1ef58937bf588cbf41688c01826843c5d6d8e4b